test-utils/runtime-tester/src/run_test.rs

```diff
@@ -89,17 +92,24 @@ pub struct TransactionConfig {
     pub actions: Vec<Action>,
 }
 
-#[derive(Serialize, Deserialize, Default)]
+#[derive(Serialize, Deserialize, Default, Debug)]
 pub struct RuntimeStats {
     pub blocks_stats: Vec<BlockStats>,
 }
 
-#[derive(Serialize, Deserialize, Default)]
+#[derive(Serialize, Deserialize, Default, Debug)]
 pub struct BlockStats {
     pub height: u64,
     pub block_production_time: Duration,
 }
 
+use std::fmt::Debug;
+impl Debug for Scenario {
```

Comment:
Nit:

```suggestion
impl fmt::Debug for Scenario {
```

we don't usually put `use` in the middle of the file